[2024-03-11T18:27:13.254+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: PythonOperator.Task1 manual__2024-03-11T18:27:11.694386+00:00 [queued]>
[2024-03-11T18:27:13.277+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: PythonOperator.Task1 manual__2024-03-11T18:27:11.694386+00:00 [queued]>
[2024-03-11T18:27:13.279+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-11T18:27:13.313+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): Task1> on 2024-03-11 18:27:11.694386+00:00
[2024-03-11T18:27:13.320+0000] {standard_task_runner.py:60} INFO - Started process 4215 to run task
[2024-03-11T18:27:13.328+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'PythonOperator', 'Task1', 'manual__2024-03-11T18:27:11.694386+00:00', '--job-id', '627', '--raw', '--subdir', 'DAGS_FOLDER/2_python_operators.py', '--cfg-path', '/tmp/tmp0hmaf1i_']
[2024-03-11T18:27:13.331+0000] {standard_task_runner.py:88} INFO - Job 627: Subtask Task1
[2024-03-11T18:27:13.447+0000] {task_command.py:423} INFO - Running <TaskInstance: PythonOperator.Task1 manual__2024-03-11T18:27:11.694386+00:00 [running]> on host 482398f6ce07
[2024-03-11T18:27:13.568+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='PythonOperator' AIRFLOW_CTX_TASK_ID='Task1' AIRFLOW_CTX_EXECUTION_DATE='2024-03-11T18:27:11.694386+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-11T18:27:11.694386+00:00'
[2024-03-11T18:27:13.572+0000] {logging_mixin.py:188} INFO - Hola desde el PythonOperator
[2024-03-11T18:27:13.574+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-11T18:27:13.594+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=PythonOperator, task_id=Task1, execution_date=20240311T182711, start_date=20240311T182713, end_date=20240311T182713
[2024-03-11T18:27:13.624+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-11T18:27:13.653+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
